### PR TITLE
Add Pycom's Pygate to the list of Gateways

### PR DESCRIPTION
**Pull request by danicampora**
Just description and specs for now. Setup instructions and guidelines will come later before the Gateway starts shipping.

### CODE REVIEW

File: _content/gateways/pycom/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Pygate description
```
Comment (johanstokking) [inline _content/gateways/pycom/index.md:1]: Please follow the header format so this page gets rendered correctly.

For example:

```
---
title: Pycom Pygate
section: Hardware
redirect_from:
 - /pycom/
image: /gateways/pycom/image.jpg
zindex: 900
---
```

You may also need to change this header.

**Review by johanstokking — CHANGES_REQUESTED** [review]
Thanks! Please also add installation instructions for TTN/TTI.

**Comment by danicampora** [comment]
@johanstokking thanks for the feedback. I will follow the header format as you suggest. Regarding the installation instructions, that will come later as we haven't prepared that yet (the Gateway is still not in production). Can we add the installation instructions later on?

**Comment by johanstokking** [comment]
@danicampora since this is documentation, we need instructions here. For showing a product page only, we have the [Marketplace](https://www.thethingsnetwork.org/marketplace). cc @rish-c

**Comment by johanstokking** [comment]
@danicampora what is the status of the gateway production? Any manuals yet?

**Comment by benolayinka** [comment]
Closing due to inactivity

**Pull request closed without merging**